libs/wingc/src/jsify.rs

```diff
@@ -62,6 +62,8 @@ impl<'a> JSifier<'a> {
 
 	fn js_resolve_path(path_name: &str) -> String {
 		format!("require('path').resolve(__dirname, \"{}\")", path_name)
+			// escape backslashes for windows paths
```

Comment:
Just verifying: do we want to limit this to windows only? I'm assuming this is ok on linux/macOS, right?

Reply:
Technically posix does allow `\` in paths, so let me see if I can do this a little differently